distro/tests/nix-commands/configuration.nix

```diff
@@ -0,0 +1,5 @@
+{ config, lib, pkgs, ... }:
```

Comment:
Rename test-specific configuration files to `extra_config.nix` to highlight the fact that it is not a complete `configuration.nix`.